JournalCoreData/View Controllers/EntryDetailViewController.swift

```diff
@@ -41,7 +41,8 @@ class EntryDetailViewController: UIViewController {
     }
     
     private func updateViews() {
-        guard let entry = entry else {
+        guard let entry = entry,
+        isViewLoaded == true else {
```

Comment:
There is no need to say ` guard let entry = entry,
	        isViewLoaded == true else {` isViewLoaded == true. you can just say ` guard let entry = entry,
	        isViewLoaded  else {` and it does that same thing.